### ISSUE TITLE
Questions about the results on Clothing1M

### ISSUE THREAD

**Issue by JackYFL** [uitrbn/TSCSI_IDN#2]
Thank you for your open-sourced codes! However, I couldn't obtain the 75.40% result after running the codes (with the default hyper-parameters in the codes), and the result via my implementation is about 74.54%.  Do I miss something important? Looking forward to your reply!

**Comment by uitrbn** [comment]
Hi. The performance is inconsistent because I pretrained the warmup stage and loaded the checkpoint to save the training time. I retrained the model using the current script with seed 123 and got a performance of 75.52%. Maybe you could try this seed. 
I also upload the conda environment for reference.

**Comment by JackYFL** [comment]
Thank you for your reply, and I reimplement the result of 75.5% using the seed 123.

**Comment by JackYFL** [comment]
Thank you for your reply, and I reimplement the result of 75.5% using the seed 123.
